docs/builders/iso.mdx

```diff
@@ -399,6 +399,12 @@ or responding to pattern `/dev/.+`. Example:
 
 - `machine` - (string) - Set the machine type. Supported values are 'pc' or 'q35'.
 
+@include 'builder/proxmox/common/rng0Config.mdx'
```

Comment:
Regarding the documentation here, I'd suggest we move the details regarding `rng0` to a subsection in here, for reference, here's what the generated docs look like with the current code:
![image](https://github.com/hashicorp/packer-plugin-proxmox/assets/105649352/5c9e59b2-feff-4dac-86df-b629133a7617)

My only grief with this is that it's not immediately clear that the `rng0` attributes are available as part of the block here.
So I would encourage you to add a small description of the `rng0` block in this section, and reference the newly created section through a link with an anchor.

If you want to preview locally the documentation, please refer to the documentation: https://developer.hashicorp.com/packer/docs/plugins/creation#registering-plugin-documentation to build the `docs.zip` file, and the next section to locally build and see the docs.

Again, if you need a hand in organizing those, feel free to let me know and I can iterate on this.